### ISSUE TITLE
Exception when importing listings with WP All Import

### ISSUE THREAD

**Issue by trey8611** [Automattic/WP-Job-Manager#1773]
Hey guys, I'm Trey Mills and I work on WP All Import/Export for Soflyy. We recently had a customer report the following bug.

#### Steps to Reproduce
1. Install [WP All Import](https://wordpress.org/plugins/wp-all-import/), [WP Job Manager](https://wordpress.org/plugins/wp-job-manager/), and the [WP Job Manager Add-On for WP All Import](https://wordpress.org/plugins/wp-job-manager-xml-csv-listings-import/).
2. Run an import into "Jobs" using any dummy data.
3. Run the import.

#### What I Expected
Job listings to be imported.

#### What Happened Instead
The import was terminated: https://d.pr/i/ojV33j. Here's the error that was logged:

```log
[05-Jun-2019 17:47:16 UTC] Backtrace from Exception "Too few arguments to function WP_Job_Manager_Post_Types::maybe_add_default_meta_data(), 1 passed in C:\laragon\www\wpjobmanager\wp-content\plugins\wp-job-manager\includes\3rd-party\wp-all-import.php on line 17 and exactly 2 expected"In C:\laragon\www\wpjobmanager\wp-content\plugins\wp-job-manager\includes\class-wp-job-manager-post-types.php [line 1011]C:\laragon\www\wpjobmanager\wp-content\plugins\wp-job-manager\includes\3rd-party\wp-all-import.php 17 calling maybe_add_default_meta_data() | C:\laragon\www\wpjobmanager\wp-includes\class-wp-hook.php 288 calling wpjm_pmxi_saved_post() | C:\laragon\www\wpjobmanager\wp-includes\class-wp-hook.php 310 calling apply_filters() | C:\laragon\www\wpjobmanager\wp-includes\plugin.php 465 calling do_action() | C:\laragon\www\wpjobmanager\wp-content\plugins\wp-all-import\models\import\record.php 2847 calling do_action() | C:\laragon\www\wpjobmanager\wp-content\plugins\wp-all-import\controllers\admin\import.php 2577 calling process() | C:\laragon\www\wpjobmanager\wp-content\plugins\wp-all-import\plugin.php 591 calling process() | C:\laragon\www\wpjobmanager\wp-includes\class-wp-hook.php 286 calling adminInit() | C:\laragon\www\wpjobmanager\wp-includes\class-wp-hook.php 310 calling apply_filters() | C:\laragon\www\wpjobmanager\wp-includes\plugin.php 465 calling do_action() | C:\laragon\www\wpjobmanager\wp-admin\admin.php 169 calling do_action()
```

#### WordPress / WP Job Manager Version
- WordPress Version 5.2.1
- WPJM Version 1.33.0

#### Screenshot / Video
- Screenshot: https://d.pr/i/ojV33j
- Video: https://d.pr/v/vIU9AA

#### Context / Source

I am a part of the support and development team for WP All Import and this issue was reported by a customer.

**Comment by jom** [comment]
Hi @trey8611! I just put up a PR with a fix in #1773.

**Comment by trey8611** [comment]
@jom Awesome, thank you!
